.github/workflows/deploy-test.yml

```diff
@@ -0,0 +1,39 @@
+name: CD
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, closed]
```

Comment:
https://docs.github.com/ko/actions/using-workflows/events-that-trigger-workflows#pull_request